packages/peregrine/package.json

```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/magento-research/peregrine"
   },
   "scripts": {
-    "build": "babel src -d dist --ignore test.js",
+    "build": "babel src -d dist --ignore '*.test.js,*.spec.js'",
```

Comment:
No need to build the test files for distributing. But I also caught this generating built test artifacts, so jest would run all tests twice. Random tech debt from monorepo move

Reply:
We were using `.spec.js` for most of initial development, IIRC. When did we start having `.test.js` files, and why? Should change them.

Reply:
> When did we start having .test.js files

[October 31, 2017](https://github.com/magento-research/peregrine/commit/531a1deb36c0f35a17dfe68688f9003a5f58d4b3). We can change it in a separate PR